### ISSUE TITLE
Feature request: Option for custom (or no) timeout

### ISSUE THREAD

**Issue by GrandyB** [SiegeEngineers/aoe2cm2#96]
Currently, players have 30 seconds.

I am considering using aoe2cm for a LAN broadcast, and in this case I may want to have more time to report the players' picks and bans. I figure it could make sense for an option to be added to the preset creation that would allow me to either disable the timeout, or set it to a longer custom amount.

**Comment by HSZemi** [comment]
No.

Sounds harsh, but I am of course going to explain the reasoning for this.

There is one thing aoe2cm has been designed to do, and it is to force players to decide on their picks within a reasonably short amount of time. The secondary purpose is to provide a convenient way to draft remotely, and the third purpose is to provide a record about drafts.

Disabling the timeout obviously defeats the primary purpose of the tool.

Also, if you need more time to talk about choices, it sounds like you rather need a minimum wait time instead of an extension of the timeout. Which, well.

If you're covering a LAN event anyway, may I recommend you do the drafting differently? There is [excellent precedence](https://youtu.be/2u3HupyXyKQ?t=458).

**Comment by GrandyB** [comment]
> Disabling the timeout obviously defeats the primary purpose of the tool.

I agree that the primary use-case should always be to provide quick/remote/stored drafts.

I guess the more edge cases that are accommodated for, the more noise/clutter there might be in places like preset creation (and the code), and that could detract from the primary use-case... would definitely understand if that was the deeper concern with something like this.

> If you're covering a LAN event anyway, may I recommend you do the drafting differently? There is excellent precedence.

I've made something custom before, but being able to directly use (or indirectly pull from) an aoe2cm draft would make my life easier, so I'm playing around with it and seeing what does/doesn't work for my use-case.
